%YAML:1.0
vocPath: "/home/eric/slam/ORB_SLAM2/Vocabulary/ORBvoc.txt"
settingPath: "/home/eric/slam/ORB_SLAM2/Examples/Stereo/KITTI03.yaml"
do_rectify: "false"